app/(auth)/uuid.ts

```diff
@@ -0,0 +1,12 @@
+import { cookies } from 'next/headers'
+import type { Session } from '@supabase/supabase-js'
+import type { NextResponse } from 'next/server'
+
+const cookieName = 'user_uuid'
+
+export const getUuid = () => cookies().get(cookieName)?.value ?? undefined
+
+export const setUuid = (res: NextResponse, session: Session): NextResponse => {
+  res.cookies.set(cookieName, session.user.id)
```

Comment:
Get uuid from session. 
See: https://github.com/tabi-memo/backend/pull/12#issuecomment-1861915696

Reply:
Is `session.user.id` equal to the `auth.users.uuid` ?